### PR TITLE
Implement discussed optimizations

### PR DESCRIPTION
**Pull request by rs22**
Use faster random generator
Always generate 100 cols
Generate random row indices for each benchmark iteration

### CODE REVIEW

File: src/main.cpp [hunk 1]
```diff
@@ -2,18 +2,18 @@
...
 #include "column.h"
 #include "TableGenerator.h"
+#include "random.h"
 
-static auto rows = 1 * 1000 * 1000UL;
+static unsigned long rows = 1 * 1000 * 1000UL;
 static unsigned int max_cell_value = 1000000;
-static unsigned int num_of_local_columns = 50;
-static unsigned int num_of_remote_columns = 50;
+static unsigned int total_columns = 50;
```
Comment (StarkRobert) [inline src/main.cpp]: shouldnt this be 100, if we commit message says tables always have 100cols

**Review by StarkRobert — COMMENTED** [review]
Where can i see that 100 cols are always generated in a table
everything else looks great 👍

File: src/TableGenerator.cpp [hunk 2]
```diff
@@ -1,20 +1,21 @@
...
         for (unsigned long j = 0; j < numOfRows; ++j) {
-            uint32_t cellValue = (uint32_t) rand() % maxRandomNumberInCell;
+            uint32_t cellValue = (uint32_t) Random::next() % maxRandomNumberInCell;
             column.get()->data().at(j) = cellValue;
         }
         table.addColumn(column);
     }
     for (unsigned int i = 0; i < numOfRemoteColumns; ++i) {
-        auto column = std::make_shared<Column<uint32_t>>(numOfRows, numa_max_node());
+        auto column = std::make_shared<Column<uint32_t>>(numOfRows, 3);
```
Comment (StarkRobert) [inline src/TableGenerator.cpp]: any reason why node=3 or is this just randomly chosen?

**Comment by rs22** [comment]
Fixed the total_columns variable; thanks!

**Review by StarkRobert — APPROVED** [review]

File: src/TableGenerator.cpp (hunk 2 above)
Comment (rs22) [reply src/TableGenerator.cpp]: Sorry, I've reverted it -- I didn't mean to check this in, was only testing...

**Review by NHoff95 — APPROVED** [review]

**Pull request merged**